src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect.java

```diff
@@ -9,52 +9,66 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparagraph;
 
 /**
- * Some Javadoc. // 2 violations below
+ * Some Javadoc. // violation '\Q Javadoc content should start from the next line
+ * after /\*\*.\E'
```

Comment:
Why you add a extra content in input file?

If 2 violation on same line happened we do not validate message, as we not know where which to choose.

Reply:
You mean,  when we have `// 2 violations` or `// 2 violation below` or `// 2 violation above` then we should not validate a message, right?

Reply:
Yes, you can go through https://github.com/checkstyle/checkstyle/issues/11214#issue-1106705357 once again, it mentions that, also you can go through other similar PR's, they will help you in making changes.

Reply:
Can we do:
```
// Violation 2 lines below, `message 1`
// Violation below, `message 2`
```
?

Reply:
@romani I believe we can't do that as per the changes suggested by @Vyom-Yadav at https://github.com/checkstyle/checkstyle/issues/11214#issue-1106705357

Reply:
did you try ?
I do not see issue that claim that it is not possible.

Reply:
I tried to do it and it happened. You were right.